dist/LICENSE

```diff
@@ -214,33 +214,491 @@ Apache 2.0 licenses
 
 The following components are provided under the Apache License. See project link for details.
 The text of each license is the standard Apache 2.0 license.
-    client_golang (prometheus) v1.9.0: https://github.com/prometheus/client_golang Apache 2.0
-    grpc-go (grpc) v1.36.1: https://github.com/grpc/grpc-go Apache 2.0
-    prometheus (prometheus) v1.8.2: http://github.com/prometheus/prometheus Apache 2.0
-    go-yaml (yaml) v3.0.0: https://github.com/go-yaml/yaml Apache 2.0
+
+    cloud.google.com-go v0.26.0 https://cloud.google.com/go Apache-2.0
```

Comment:
All following are standard Apache 2.0 license. We don't need a copy of the license file. Please remove them.

Reply:
Deleted.